### PR TITLE
Use Markdown for docs

### PR DESCRIPTION
**Pull request by vprithvi**
via mkdoc

### CODE REVIEW

**Comment by vprithvi** [comment]
It looks like http://jaeger-docs.readthedocs.io/en/docs/

File: docs/getting_started/tracing_sample_app.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+-   Discover architecture of the whole system via data-driven dependency
+    diagram
+-   View request timeline & errors, understand how the app works
+-   Find sources of latency, lack of concurrency
+-   Highly contextualized logging
+-   Use baggage propagation to
+
+    -   Diagnose inter-request contention (queueing)
+
+    \* Attribute time spent in a service
```
Comment (yurishkuro) [inline docs/getting_started/tracing_sample_app.md]: what is `\*` ?

File: docs/getting_started/tracing_sample_app.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+-   Use open source libraries with OpenTracing integration to get
+    vendor-neutral instrumentation for free
+
+### Prerequisites
+
+-   You need Go 1.7 or higher installed on your machine.
+-   Requires a running Jaeger backend to view the traces.
+    -   See backend\_in\_docker
```
Comment (yurishkuro) [inline docs/getting_started/tracing_sample_app.md]: ? this won't work as a link

**Comment by yurishkuro** [comment]
> It looks like http://jaeger-docs.readthedocs.io/en/docs/

how did it get there?

**Comment by yurishkuro** [comment]
please add instructions how to build & test this locally, and how to publish

**Comment by yurishkuro** [comment]
I would like these to still be used:
* docs/_static/favicon.ico
* docs/_static/logo_ng_200.jpg

**Comment by vprithvi** [comment]
@yurishkuro I'm rewriting the docs in the next PR, and will make sure your comments are addressed.

**Pull request merged**